.github/workflows/slash-commands.yml

```diff
@@ -23,7 +27,8 @@ jobs:
             gke-kube-test
             run-specific-test
           static-args: |
-            ref=${{ steps.getref.outputs.ref }}
+            repo=${{ steps.getref.outputs.repo }}
+            gitref=${{ steps.getref.outputs.ref }}
```

Comment:
:information_source: The `ref` keyword was a special keyword, running the corresponding action against that branch. But this only works if the branch was on the upstream repository and not a fork. Since it now can also be on a fork, we just run the action always against master (i.e. the action code will be taken from master). Thus we now pass in `repo` and `gitref` explicitly as input parameters to the command actions, which will then set this on all `checkout` actions.